arduino-ide-extension/src/browser/contributions/verify-sketch.ts

```diff
@@ -18,15 +18,21 @@ export class VerifySketch extends SketchContribution {
     @inject(BoardsServiceProvider)
     protected readonly boardsServiceClientImpl: BoardsServiceProvider;
 
+    verifyInProgress = false;
```

Comment:
This should be a `protected` field, not a `public` mutable property. Same for the upload part.